### ISSUE TITLE
Drillthrough Dodgy

### ISSUE THREAD

**Issue by buggtb** [OSBI/saiku#645]
often get array out of bounds exceptions

**Comment by OliverFranco** [comment]
Yes, I have see this error. There are a Bug in the function "drillthroughWithCaptions".  Also, there are many errors with the columns name of the Drillthrough. I modified this class, I add 2 .class, with the solutions.

[saiku_service_Ciber.zip](https://github.com/OSBI/saiku/files/3819595/saiku_service_Ciber.zip)
